Configuration/AlCa/python/autoAlca.py

```diff
@@ -51,7 +51,7 @@ def buildList(pdList, matrix):
 
 # Update the lists anytime a new PD is added to the matrix
 autoAlca = { 'allForPrompt'         : buildList(['Charmonium', 'Commissioning', 'DoubleMuParked', 'DoubleMuon', 'DoubleMuonLowMass', 'EGamma', 'HLTPhysics', 'HcalNZS', 'JetHT', 'MET', 'MinimumBias', 'MuOnia', 'MuOniaParked', 'NoBPTX', 'SingleMuon', 'ZeroBias'], AlCaRecoMatrix),
```

Comment:
Not relevant for this PR, but I am noticing that the 'allForPrompt' list is differenn here and in the master or 12_5_X
@tvami is it intended? (Perhaps I just fogot when it branched away...)

Reply:
@tvam (or @cms-sw/alca-l2) could you please check and answer this, before it gets forgotten, fiven that the PR is now merged?

Reply:
hi @perrotta sorry for the delay! we'll take a look asap! 
(wrong Tamas tagged btw :D )

Reply:
Hi @perrotta , this should indeed have been backported from PR #38951 . I will submit the backport today. Thank you very much for spotting it!